### PR TITLE
Update LUT API settings

### PR DESCRIPTION
**Pull request by edmondchuc**
This update fixes and completes the current set of LUT API settings.

It also makes some minor changes to:
- `.gitignore`
- dataclasses are now immutable

### CODE REVIEW

File: dawe_vocabs/settings.py [hunk 1]
```diff
@@ -27,441 +27,1450 @@
...
     ),
     LUTSchema(
-        "http://vocabs.paratoo.tern.org.au:1337/api/lut-voucher-types",
-        "Voucher type codes",
-        "A collection of voucher types and its codes.",
-        "a3333793-95bd-47fc-ba90-b8fc65a40456",
-        "voucher type",
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-fauna-pouch-conditions",
+        label="Animal pouch condition codes",
+        description="A collection of animal pouch condition types and its " "codes.",
```
Comment (junrongYu) [inline dawe_vocabs/settings.py:46]: hi Edmond, is the extra " syntax error?

Comment (edmondchuc) [reply dawe_vocabs/settings.py:46]: I think that is due to black's auto-formatting behaviour. In any case, it's not a syntax error. Two strings together performs a concatenation into one string.

File: dawe_vocabs/settings.py [hunk 2]
```diff
@@ -27,441 +27,1450 @@
...
+    LUTSchema(
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-taxa-types",
+        label="Taxa type codes",
+        description="A collection of taxa types and its codes.",
+        collection_uuid="7ea12fed-6b87-4c20-9ab4-600b32ce15ec",
+        uuid_namespace="taxa type",
+    ),
+    LUTSchema(
+        endpoint_url="http://vocabs.paratoo.tern.org.au:1337/api/lut-tecs",
+        label="Tecs codes",
```
Comment (junrongYu) [inline dawe_vocabs/settings.py:1324]: can we expand `tecs` to `threatened ecological communities`?

Comment (edmondchuc) [reply dawe_vocabs/settings.py]: Done.

**Review by junrongYu — APPROVED** [review]

**Pull request merged**